resolwe/permissions/utils.py

```diff
@@ -101,6 +102,15 @@ def update_permission(obj, data):
     content_type = ContentType.objects.get_for_model(obj)
     full_permissions = list(zip(*obj._meta.permissions))[0]  # pylint: disable=protected-access
 
+    def apply_perm(perms, entity, perm_func):
```

Comment:
Add documentation block explaining what the function does and what parameters it should be given.